packages/sdk/src/FlagResolverClient.ts

```diff
@@ -292,7 +292,7 @@ export function withRequestLogic(fetchImplementation: (request: Request) => Prom
     // update send-time before sending
     .modifyRequest(async request => {
       if (request.method === 'POST') {
-        const body = JSON.stringify({ ...(await request.json()), sendTime: new Date().toISOString() });
+        const body = JSON.stringify({ ...(await request.clone().json()), sendTime: new Date().toISOString() });
```

Comment:
We need to clone the request before reading the payload. Otherwise creating a new request in the next line will fail because it thinks the request is already consumed. Most implementations are more lenient, because we override with a new body so it doesn't matter that the previous body was consumed.